pandas/tools/pivot.py

```diff
@@ -106,6 +106,10 @@ def pivot_table(data, values=None, index=None, columns=None, aggfunc='mean',
         else:
             values_multi = False
             values = [values]
+        # GH14938 Make sure values are in data
```

Comment:
blank line before the comment.

say valuess indexer (or labels) (to clear these are not actual values, but the indexers of the labels)